cove_iati/lib/iati.py

```diff
@@ -447,3 +479,93 @@ def organisation_identifier_count(tree):
     unique_identifiers = list(set(identifiers))
 
     return len(unique_identifiers)
+
+
+ACTIVITY_PREFIX = '/iati-activities/iati-activity'
```

Comment:
Have we thought about whether we want to check org files too?

Reply:
yes but that can be at a seperate time.  It requires checking different fields.